### ISSUE TITLE
Docs update:  SimpleSAMLphp SP API reference 

### ISSUE THREAD

**Issue by monkeyiq** [simplesamlphp/simplesamlphp#2275]
The docs for SP API reference mention the name of the SP to use as `string $authSource`

```
It has a single parameter, which is the ID of the authentication source that should be used. 
This authentication source must exist in config/authsources.php . 
```

For SSP 2.x this has to be a `saml:SP` or you will likely see the error message "Authsource must be of type saml:SP." when you try to use things.

On a related topic, I think for new users it might be handy to also link here to a basic setup

`SP -> IdP with metadata.auth -> authsource`
 
To give a hint of how a minimal recommended local auth setup might be created. For example, using a relational database as the authsource how they might get the SP/IdP setup going to allow a small 1-30 user setup to work.

**Comment by monkeyiq** [comment]
I have made a reference to sspsmall there. To those reading this, sspsmall is pre alpha, and mostly just for fun for me at the moment. I have found it useful as a test platform for some PRs though so it might become a more main project as time goes by. Handy to be able to test and verify issues with folks who have much more advanced SSP configurations.

Right now sspsmall is at https://github.com/monkeyiq/sspsmall  I am happy for it to move to the ssp repo once it is more stable and tested.
